redbot/core/core_commands.py

```diff
@@ -1687,10 +1687,11 @@ class Core(commands.Cog, CoreLogic):
         pass
 
     @whitelist.command(name="add")
-    async def whitelist_add(self, ctx, user: discord.User):
+    async def whitelist_add(self, ctx, *, user: Union[discord.Member, int]):
         """
         Adds a user to the whitelist.
         """
+        uid = getattr(user, "id", user)
```

Comment:
Any reason why these changes aren't being added to `[p]blacklist` as well?

Reply:
Intended for both, updated.